apps/job-service/src/jobs/delayed-jobs.controller.ts

```diff
@@ -97,7 +97,20 @@ export class DelayedJobsController {
         uuid: { [Op.in]: jobUpdates.map(({ uuid }) => uuid) },
         createdBy: authenticatedUserId,
         status: { [Op.ne]: "pending" }
-      }
+      },
+      include: [
+        {
+          association: "entityProject",
+          attributes: ["name"],
+          required: false
+        },
+        {
+          association: "entitySite",
+          attributes: ["name"],
+          required: false
+        }
+      ],
+      logging: console.log
```

Comment:
This should not be necessary. At least in both my local and AWS, query logging is already functional.